eng/Versions.props

```diff
@@ -21,6 +21,7 @@
 
     <!-- Roslyn -->
     <MicrosoftCodeAnalysisVersion>3.0.0</MicrosoftCodeAnalysisVersion>
+    <MicrosoftCodeAnalysisForShippedApisVersion>3.5.0-beta2-20056-01</MicrosoftCodeAnalysisForShippedApisVersion>
```

Comment:
@mavasani @sharwell I've update this PR to use the latest compiler package for testing (that package supports the new display option for `!`), but now I can't run tests any more (I tried the tests for the new analyzer, as well as an existing one).

Let me know if you have any clue what might be going on. I don't see the possible link between using a new compiler package and hitting a roslyn-SDK nuspec issue with packages for reference assemblies.

The error  is:
```
 Message: 
    NuGet.Packaging.Core.PackagingException : The package is missing the required nuspec file. Path: C:\Users\jcouv\AppData\Local\Temp\test-packages\Microsoft.NETFramework.ReferenceAssemblies.net472.1.0.0-preview.2
  Stack Trace: 
    PackageFolderReader.GetNuspecFile()
    PackageReaderBase.GetNuspec()
    PackageReaderBase.get_NuspecReader()
    PackageReaderBase.GetFrameworkItems()
    PackageReaderBase.GetFrameworkItemsAsync(CancellationToken cancellationToken)
    <ResolveCoreAsync>d__39.MoveNext() line 316
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    TaskAwaiter`1.GetResult()
    <ResolveAsync>d__38.MoveNext() line 168
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    TaskAwaiter`1.GetResult()
    <CreateSolutionAsync>d__71.MoveNext() line 692
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    TaskAwaiter`1.GetResult()
    <CreateProjectImplAsync>d__70.MoveNext() line 625
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    TaskAwaiter`1.GetResult()
    <CreateProjectAsync>d__69.MoveNext() line 603
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    TaskAwaiter`1.GetResult()
    <GetSolutionAsync>d__68.MoveNext() line 577
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    TaskAwaiter`1.GetResult()
    <GetSortedDiagnosticsAsync>d__65.MoveNext() line 491
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    ConfiguredTaskAwaiter.GetResult()
    <VerifyDiagnosticsAsync>d__55.MoveNext() line 197
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    ConfiguredTaskAwaiter.GetResult()
    <RunAsync>d__38.MoveNext() line 189
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    TaskAwaiter.GetResult()
    <VerifyAdditionalFileFixAsync>d__2.MoveNext() line 59
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    TaskAwaiter.GetResult()
    <VerifyCSharpAdditionalFileFixAsync>d__1.MoveNext() line 43
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    TaskAwaiter.GetResult()
    <AnnotatedMemberInAnnotatedShippedAPI>d__5.MoveNext() line 135
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    --- End of stack trace from previous location where exception was thrown ---
    TaskAwaiter.ThrowForNonSuccess(Task task)
    TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
```

Reply:
Never mind. I deleted the local temp folder and retried. Tests are working again now :-)